openlibrary/templates/books/edition-sort.html

```diff
@@ -64,7 +64,7 @@
           <!-- Formats are store as a raw string -->
           $book.physical_format.replace('[', '').replace(']','')
           $if book.languages:
-            $:_('in %(languagelist)s', languagelist=commify_list(thingrepr(l) for l in book.languages))
+            $:_('in %(languagelist)s', languagelist=commify_list(websafe(l.name) for l in book.languages))
```

Comment:
Oh actually also just noticed this changes the i18n, since the i18n was being done in `thingrepr` (defined in templates/languages/repr.html)! So we need to manually call get_language_name.
```suggestion
            $:_('in %(languagelist)s', languagelist=commify_list(websafe(get_language_name(l.key)) for l in book.languages))
```